src/protocol/graphics.ts

```diff
@@ -515,10 +515,9 @@ export function refreshGraphics() {
       cx.drawImage(image, 0, 0);
     }
 
-    onRefreshGraphics({
-      ...bounds,
-      gDevicePixelRatio,
-    });
+    if (onRefreshGraphics) {
+      onRefreshGraphics({ ...bounds, gDevicePixelRatio });
```

Comment:
Is this a place for `onRefreshGraphics?.({ ...bounds, gDevicePixelRatio });`?

Reply:
yep!